libs/platform/src/lib/components/wizard-generator/components/dialog-wizard-generator/dialog-wizard-generator.component.scss

```diff
@@ -0,0 +1,46 @@
+.fdp-wizard-dialog {
+
+    .fd-message-strip {
+        margin-bottom: 12px !important;
+    }
+
+    .fd-dialog {
+        &__header {
+            .fd-bar {
+                box-shadow: none;
+            }
+        }
+    }
+
+    .fd-title {
+        &--h3 {
+            margin-bottom: 12px !important;
+        }
+
+        &--h2 {
+            padding: 16px 0 8px 16px;
+        }
+    }
+
+    .fd-wizard {
+        &__content {
+            padding-left: 32px;
+            padding-right: 32px;
+        }
+
+        &__progress-bar {
+            padding-left: 32px;
+            padding-right: 32px;
+        }
+    }
+
+    .fd-bar {
+        &--floating-footer {
+            margin: 8px;
+        }
+
+        &--header {
+            box-shadow: none;
+        }
+    }
```

Comment:
You are not allowed to have custom styling in ngx, esp overwriting fundamental-styes classes with values in px